spark/v3.0/spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -57,6 +57,9 @@ private SparkReadOptions() {
   // Set ID that is used to fetch file scan tasks
   public static final String FILE_SCAN_TASK_SET_ID = "file-scan-task-set-id";
 
+  // Set the target task size of a file scan combined tasks
+  public static final String FILE_SCAN_TARGET_SIZE = "file-scan-target-size";
```

Comment:
The name is not very descriptive, to be honest. It starts with `file-scan`, which makes it specific to `SparkFilesScan`, like it should but it is not clear what `target-size` means. I think it defines the target task size whenever reading file scan task sets.

I don't really have a good alternative to suggest and it is an internal option, so I probably wouldn't worry.

Reply:
It is internal so I don't really want to worry about it too much